.browserslistrc

```diff
@@ -0,0 +1,13 @@
+# Browsers that we support
+
+[production]
+>0.2% and supports es6-class
+not dead
+not op_mini all
+not chrome < 51
+not safari < 10
```

Comment:
That not the rules in `package.json`
```suggestion
>0.2%
not dead
not op_mini all
```

Reply:
yes, not all fronts are sync, that another work to do, rules aren't in sync with our production env

Reply:
Why we don't use the same ? Is there a problem with the config from gridstudy on gridadmin ?

Reply:
okay